src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlInternalConnectionTds.cs

```diff
@@ -1075,9 +1075,11 @@ private void ResetConnection()
                 // distributed transaction - otherwise don't reset!
                 if (Is2000)
```

Comment:
Leaving Is2000 logic in place for now until we remove support for it. The fix will not apply to versions before 2000.